solutions/devsprint-bruno-ramos-2/DeliveryAppChallenge/Modules/Product/Home/HomeInteractor.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  HomeInteractor.swift
+//  DeliveryAppChallenge
+//
+//  Created by Leonardo Mendes on 30/01/22.
+//
+
+import Foundation
+
+/// Protocol used to intermediate communication from ViewController to Interactor ( ViewController ----`request` ----> Interactor )
+protocol HomeBusinessLogic {
+    func requestData(request: HomeUseCase.FetchData.Request)
```

Comment:
nesse caso, seguindo a nomenclatura do VIP, podemos alterar o nome do método para explicitar quais dados ele está pegando.
```suggestion
    func fetchRestaurantList(request: HomeUseCase.FetchData.Request)
```

Reply:
Corrigido